### PR TITLE
docs: prioritize ng add in installation guides

### LINKED ISSUE #3434: Installation docs
### Information

All of the docs have installation instructions for `npm`, `yarn`, and `ng add`.
Currently, `ng-add` is added lastly and we have the following guide for it:

```md
If your project is using the Angular CLI 6+ then you can install 
the plugin to your project with the following `ng add` command
```

I would propose to move `ng-add` to the top because some `ng add` commands have some useful stuff included, and it's also a common command now. We can also rephrase the text, for example we can remove "If your project is using the Angular CLI 6+".

### Documentation page

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by mauromattos00**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

- The `ng add` command, which automatically makes useful changes to files related to the lib installation, is being shown after the `npm install` and `yarn install` commands.
- The optional flags are displayed as a list.

Closes #3434

## What is the new behavior?

- The `ng add` command is the first option in the installation guides.
- The optional flags are displayed in a markdown datatable.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### Screenshot example

ng-add

### CODE REVIEW

File: projects/ngrx.io/content/guide/eslint-plugin/install.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Installation
 
+## Installing with `ng add`
+
+You can install the plugin to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/eslint-plugin/install.md:5]:
```suggestion
You can install the ESLint plugin to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
```

File: projects/ngrx.io/content/guide/component-store/install.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Installation
 
+## Installing with `ng add` (recommended)
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/install.md:3]: To keep it consistent with the other docs, remove `recommended`.

```suggestion
## Installing with `ng add`
```

**Review by timdeschryver — COMMENTED** [review]
Loving the option tables! 🤩

**Review by timdeschryver — APPROVED** [review]

**Review by brandonroberts — APPROVED** [review]
LGTM 🤩

**Pull request merged**